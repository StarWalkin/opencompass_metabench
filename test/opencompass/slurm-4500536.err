  0%|          | 0/3 [00:00<?, ?it/s]                                       0%|          | 0/3 [00:00<?, ?it/s] 33%|███▎      | 1/3 [00:38<01:16, 38.19s/it]                                              33%|███▎      | 1/3 [00:38<01:16, 38.19s/it] 67%|██████▋   | 2/3 [1:03:05<36:59, 2219.91s/it]                                                  67%|██████▋   | 2/3 [1:03:05<36:59, 2219.91s/it]100%|██████████| 3/3 [1:05:02<00:00, 1259.89s/it]100%|██████████| 3/3 [1:05:02<00:00, 1300.92s/it]
  0%|          | 0/3 [00:00<?, ?it/s]                                       0%|          | 0/3 [00:00<?, ?it/s] 33%|███▎      | 1/3 [00:10<00:20, 10.44s/it]                                              33%|███▎      | 1/3 [00:10<00:20, 10.44s/it] 67%|██████▋   | 2/3 [00:19<00:09,  9.77s/it]                                              67%|██████▋   | 2/3 [00:19<00:09,  9.77s/it]100%|██████████| 3/3 [00:29<00:00,  9.55s/it]100%|██████████| 3/3 [00:29<00:00,  9.68s/it]
