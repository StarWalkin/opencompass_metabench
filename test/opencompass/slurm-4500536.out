/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
08/17 11:52:48 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/17 11:52:48 - OpenCompass - INFO - Loading hf_gemma2_9b_it: configs/models/gemma/hf_gemma2_9b_it.py
08/17 11:52:48 - OpenCompass - INFO - Loading hf_glm4_9b_chat: configs/models/chatglm/hf_glm4_9b_chat.py
08/17 11:52:48 - OpenCompass - INFO - Loading hf_phi_3_small_8k_instruct: configs/models/phi/hf_phi_3_small_8k_instruct.py
08/17 11:52:48 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/17 11:52:48 - OpenCompass - INFO - Current exp folder: outputs/default/20240817_115248
08/17 11:52:48 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/17 11:52:48 - OpenCompass - INFO - Partitioned into 3 tasks.
launch OpenICLInfer[gemma2-9b-it-hf/gsm8k] on GPU 0,1,2,3
08/17 11:53:27 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/local.py - _launch - 228 - task OpenICLInfer[gemma2-9b-it-hf/gsm8k] fail, see
outputs/default/20240817_115248/logs/infer/gemma2-9b-it-hf/gsm8k.out
launch OpenICLInfer[glm-4-9b-chat-hf/gsm8k] on GPU 0,1,2,3
launch OpenICLInfer[phi-3-small-8k-instruct-hf/gsm8k] on GPU 0,1,2,3
08/17 12:57:51 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/local.py - _launch - 228 - task OpenICLInfer[phi-3-small-8k-instruct-hf/gsm8k] fail, see
outputs/default/20240817_115248/logs/infer/phi-3-small-8k-instruct-hf/gsm8k.out
08/17 12:57:51 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/base.py - summarize - 64 - OpenICLInfer[gemma2-9b-it-hf/gsm8k] failed with code 1
08/17 12:57:51 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/base.py - summarize - 64 - OpenICLInfer[phi-3-small-8k-instruct-hf/gsm8k] failed with code 1
08/17 12:57:51 - OpenCompass - INFO - Partitioned into 3 tasks.
launch OpenICLEval[gemma2-9b-it-hf/gsm8k] on CPU 
launch OpenICLEval[glm-4-9b-chat-hf/gsm8k] on CPU 
launch OpenICLEval[phi-3-small-8k-instruct-hf/gsm8k] on CPU 
dataset    version    metric    mode    gemma2-9b-it-hf      glm-4-9b-chat-hf  phi-3-small-8k-instruct-hf
---------  ---------  --------  ------  -----------------  ------------------  ----------------------------
gsm8k      1d7fe4     accuracy  gen     -                               77.18  -
08/17 12:58:20 - OpenCompass - INFO - write summary to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_115248/summary/summary_20240817_115248.txt
08/17 12:58:20 - OpenCompass - INFO - write csv to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_115248/summary/summary_20240817_115248.csv
All processes have been started and their PIDs stored in process_ids_gpt4o.txt.
All processes have completed.
